client/lib/feed-store/index.js

```diff
@@ -89,6 +94,17 @@ function receiveFeeds( newFeeds ) {
 	} ) );
 }
 
+function discoverFeeds( data, feedUrl ) {
+	if ( typeof data !== 'undefined' ) {
+		data.feeds.map( function( feed ) {
+			if ( feed.feed_ID ) {
+				feedsByUrl[ feedUrl ] = feed.feed_ID;
```

Comment:
I actually believe `forEach` would be even better than `reduce` here (because we don't need the `accumulator` for anything). So I've swapped `map` with `forEach`.